File: 2AP4_2/python_2/formatowanie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  formatowanie.py
import math

def main(args):
    lf = 12.23456789
    print("Liczba Pi: %6.2f" % math.pi)
    print("Liczba Pi: {:6.2f}".format(math.pi))
    print(f"Liczba Pi: {math.pi}")
    
    li = 100
    print(hex(li))
    print(oct(li))
    print(bin(li))
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2AP4_2/cpp_2/formatowanie.cpp
/*
 * formatowanie.cpp
 */


#include <iostream>
#include <cmath>
using namespace std;

int main(int argc, char **argv)
{
    float lf = 12.3456789;
    cout << M_PI << endl;
    printf("Liczba Pi: %10.2f\n", M_PI);
    printf("Liczba Pi: %10.4f\n", lf);

    int li = 100;
    printf("Hex: %#x\n", li);
    printf("Oct: %#o\n", li);
    
    return 0;
}

